engineering-feedback/feedback-faq.md

```diff
@@ -0,0 +1,39 @@
+# Engineering Feedback Frequently Asked Questions (F.A.Q.)
+
+The questions below are common questions related to the feedback process. The answers are intended to help both Microsoft employees and customers.
+
+## When should I submit feedback versus creating an issue on GitHub, UserVoice, or sending an email directly to a Microsoft employee?
+
+It is appropriate to do both. As a customer or Microsoft employee, you are empowered to create an issue or submit feedback via the medium appropriate for service.
+
+In addition to an issue on GitHub, feedback on UserVoice, or a personal email, Microsoft employees in CSE should submit feedback via CSE Feedback.  In doing so, please reference the GitHub issue, UserVoice feedback, or email by including a link to the item or attaching the email.
+
+Submitting to CSE Feedback allows the CSE Feedback team to coalesce feedback across a wide range of sources, and thus create a unified case to submit to the appropriate Azure engineering team(s).
+
+## How can a customer track the status of a specific feedback item?
+
+Customers are not able to track the status of feedback submitted via CSE Feedback.  The CSE Feedback process is internal to Microsoft, and as such, available only to Microsoft employees.
```

Comment:
Maybe we should prefix it with "At this time, customers..."

Reply:
Updated.